build/rpmbuild_internal.h

```diff
@@ -49,6 +49,18 @@ struct Source * next;
 
 typedef struct Package_s * Package;
 
+typedef enum rpmParseLineType_e {
+    LINE_OTHER             =  0,
+    LINE_IF                = (1 << 0),
+    LINE_IFARCH            = (1 << 1),
+    LINE_IFNARCH           = (1 << 2),
+    LINE_IFOS              = (1 << 3),
+    LINE_IFNOS             = (1 << 4),
+    LINE_ELSE              = (1 << 5),
+    LINE_ENDIF             = (1 << 6),
+    LINE_INCLUDE           = (1 << 7),
+} rpmParseLineType;
+
```

Comment:
Is there some specific reason this is in the header instead of parseSpec.c where the sole user of these id's live? I don't think any of these are relevant outside that file, but could be mistaken of course.